### PR TITLE
Point to the new template

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Review by csarven — COMMENTED** [review]
Sure, let's do that. I've used GitHub's code editor to make a suggestion but I guess it didn't come out as a suggestion: https://github.com/solid/specification/commit/0b1dde1f7b42f363d4a13bca73d748c584551898

Any way, the idea is the same. I just thought we can mention / link in the details section.

File: meetings/template.md [hunk 1]
```diff
@@ -2,62 +2,8 @@
   <summary>Document Status</summary>
   <dl>
     <dt>Modified</dt>
-    <dd>2024-01-10</dd>
+    <dd>2024-02-26</dd>
     <dt>Reason</dt>
-    <dd>This document is no longer in effect and is no longer being updated here. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
+    <dd>This document is no longer in effect and is no longer being updated here. You can access the <a href="https://github.com/w3c-cg/solid/blob/main/meetings/template.md">effective meeting template</a> for the current version. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contributing guidelines</a>.</dd>
```
Comment (TallTed) [inline meetings/template.md:7]:
```suggestion
    <dd>This document is no longer in effect and is no longer being updated here. To add a meeting, please refer to the <a href="https://github.com/w3c-cg/solid/blob/main/meetings/template.md">current meeting template</a>. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contribution guidelines</a>.</dd>
```

File: meetings/template.md [hunk 2]
```diff
@@ -4,6 +4,6 @@
...
+    <dd>Although the meeting minutes are still in this folder, this document (the template) is no longer in effect and is no longer being updated here. To add a meeting, please refer to the <a href="https://github.com/w3c-cg/solid/blob/main/meetings/template.md">current meeting template</a>. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contribution guidelines</a>.</dd>
```
Comment (TallTed) [inline meetings/template.md:7]:
```suggestion
    <dd>Although the meeting minutes are still in this folder, this document (the template) is no longer in effect and is no longer being updated. To add a meeting, please refer to the <a href="https://github.com/w3c-cg/solid/blob/main/meetings/template.md">current meeting template</a>. For the latest information on the <a href="https://www.w3.org/groups/cg/solid/">W3C Solid Community Group</a>, please refer to its <a href="https://github.com/w3c-cg/solid/">repository</a>, <a href="https://www.w3.org/community/solid/charter/">charter</a>, and <a href="https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md">contribution guidelines</a>.</dd>
```

**Comment by michielbdejong** [comment]
> Sure, let's do that.

I'll take that as an 'approve'

**Pull request merged**